Apps/WebClient/src/ClientApp/src/components/DatePickerComponent.vue

```diff
@@ -21,13 +21,18 @@ library.add(faCalendar);
 export default class DatePickerComponent extends Vue {
     @Model("change", { type: String }) public model!: string;
     @Prop({ default: undefined }) state?: boolean;
+    @Prop({ default: undefined }) maxPickerDate?: Date;
     @Ref("datePicker") datePicker!: BFormDatepicker;
 
     private value = "";
     private inputValue = "";
+    private maxDate = new Date("2100-01-01");
 
     private mounted(): void {
         this.value = this.model;
+        if (this.maxPickerDate) {
+            this.maxDate = this.maxPickerDate;
+        }
```

Comment:
This could be simplified to:

```
@Prop({ default: new Date("2100-01-01") }) maxPickerDate: Date;
```

Nit: I'd suggest renaming the prop to `maxDate`.

Reply:
Fixed